client/components/Cart.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import {connect} from 'react-redux'
+import {Link} from 'react-router-dom'
+// import thunks from store
+
+class Cart extends React.Component {
+  componentDidMount() {}
+
+  render() {
+    const {user} = this.props
+
+    const cartItems = ['item']
+    // const {cartItems} = this.props
+
+    if (cartItems.length > 0) {
+      return (
+        <div className="page">
+          <h1>My Cart</h1>
+          <div className="cart-grid">
+            <div className="cart-item">
+              <p>item name</p>
+              <p>price</p>
+              <p>quantity</p>
+              <p>image</p>
```

Comment:
maybe me we can map over function that grabs all cart items in order to display them as cart item rows